xml/System.Xml.Linq/XElement.xml

```diff
@@ -4231,6 +4231,7 @@ orderDate=10/6/2006
         <exception cref="T:System.FormatException">The element does not contain a valid <see cref="T:System.DateTime" /> value.</exception>
         <exception cref="T:System.ArgumentNullException">The <paramref name="element" /> parameter is <see langword="null" />.</exception>
         <related type="Article" href="https://msdn.microsoft.com/library/f0fe21e9-ee43-4a55-b91a-0800e5782c13">LINQ to XML</related>
+        <exception cref="T:System.InvalidOperationException">Thrown if the specified element is <see langword="null" />.</exception>
```

Comment:
So is it the ANE on line 4232 or this IOE?

Also, the tool seems to have placed this exception after a `<related>` tag instead of before it.

Reply:
Same comments throughout the rest of this file.

Reply:
> Also, the tool seems to have placed this exception after a `<related>` tag instead of before it.

That's a bug I need to fix.

Reply:
```suggestion
```